packages/cli/src/test/integration/integration_test.ts

```diff
@@ -34,6 +37,24 @@ suite('integration tests', function() {
 
   suite('init templates', () => {
 
+    skipOnWindows('test the Polymer 3.x element template', async () => {
+      const dir =
+          await runGenerator(createElementGenerator('polymer-3.x'))
+              .withPrompts({name: 'my-element'})  // Mock the prompt answers
+              .toPromise();
+      // TODO(bicknellr): Use `polymer install` once it supports installing npm
+      // packages.
+      await exec('npm install', {cwd: dir});
+
+      // TODO(bicknellr): Add this back in when `polymer lint` has a Polymer 3
```

Comment:
Let's use test urls instead of usernames for TODOs. https://github.com/Polymer/tools/issues/130 for this one